agent/agent.go

```diff
@@ -606,7 +606,8 @@ func (a *Agent) runTask(ctx context.Context, tcInput *taskContext, nt *apimodels
 	})
 
 	tskCtx = utility.ContextWithAttributes(tskCtx, tc.taskConfig.TaskAttributes())
-	tskCtx, span := a.tracer.Start(tskCtx, fmt.Sprintf("task: '%s'", tc.taskConfig.Task.DisplayName))
+	tskCtx, span := a.tracer.Start(tskCtx, "task")
```

Comment:
do we want to do the same for the [function spans](https://github.com/evergreen-ci/evergreen/blob/26c086e9bd295bf13d87add1e9f0e5da107ddc35/agent/command.go#L187)?

Reply:
I'm torn about this. On the one hand, it seems like the correct thing to do. On the other hand, it would make viewing a trace harder: https://ui.honeycomb.io/mongodb-4b/environments/production/datasets/evergreen-agent/result/xGe4zGJjqpi/trace/477vTF73h2v?fields[]=s_name&fields[]=s_serviceName&span=36e4a995c90acfeb. On the other hand, you can add the fields as a column.

I made the change though. Maybe I can ask in Slack before I merge?

Reply:
Either way's good by me 😄